src/js/components/Drop/DropContainer.js

```diff
@@ -209,23 +209,61 @@ const DropContainer = forwardRef(
             less room in the other direction leave the Drop in its current
             position. */
           if (
+            /* The bottom of the drop is aligned to the top of the target 
+            and the top of the drop is either at the top edge of the viewport
+            or beyond it. If there is room to display the drop below the target
+            change the drop's position so that the top of the drop is aligned 
+            to the bottom of the target. */
             responsive &&
-            ((align.top === 'top' && targetRect.top < 0) ||
-              (align.bottom === 'top' &&
-                targetRect.top - containerRect.height <= 0 &&
-                targetRect.bottom + containerRect.height < windowHeight))
+            align.bottom === 'top' &&
+            targetRect.top - containerRect.height <= 0 &&
+            targetRect.bottom + containerRect.height < windowHeight
           ) {
             top = targetRect.bottom;
             maxHeight = top;
           } else if (
+            /* The top of the drop is aligned to the top of the target, the 
+            bottom of the drop is either at the bottom edge of the viewport
+            or beyond it, and the height of the drop is larger than the target. 
```

Comment:
I know we discussed it but can you remind me why it matters that the height of the drop is larger than the target?

Reply:
If the height of the drop is smaller than the target and the drop is positioned so that it is displayed over the target then it doesn't do much good to try and reposition the drop unless we had a really large target. This check just prevents unnecessary re-positioning that doesn't really help make the drop more visible